shell/circleci/setup.sh

```diff
@@ -26,18 +26,22 @@ ensure_mise_installed
 
 "$CI_DIR/env/mise.sh"
 
-authn=(
-  "npm"
-  "ssh"
-  "aws"
-  "github"
-  "github_packages"
-)
-
-for authName in "${authn[@]}"; do
-  info "🔒 Setting up $authName access"
-  "$CI_DIR/auth/$authName.sh"
-done
+if [[ -n $CIRCLE_PR_REPONAME ]]; then
```

Comment:
nit: Maybe should have a top-level `is_fork` function? Bonus points for adding it to to a library

Reply:
That's fair, I'll add that to my follow-up PR next week.